tools/mtouch/error.cs

```diff
@@ -30,7 +30,7 @@ namespace Xamarin.Bundler {
 	//					MT0019	Only one --[log|install|kill|launch]dev or --[launch|debug]sim option can be used.
 	//					MT0020	The valid options for '{0}' are '{1}'.
 	//					MT0021	Cannot compile using gcc/g++ (--use-gcc) when using the static registrar (this is the default when compiling for device). Either remove the --use-gcc flag or use the dynamic registrar (--registrar:dynamic).
-	//					MT0022  The options '--unsupported--enable-generics-in-registrar' and '--registrar' are not compatible.
+	//					MT0022  <unused>
```

Comment:
I've been using "<unused, can be reused>": https://github.com/rolfbjarne/xamarin-macios/blob/59a35b9a740be4039004b98eca3da845281a3887/tools/mtouch/error.cs#L79

We have plenty of numbers left before we start re-using them 😄

I agree about mtouch-errors.md though.